Examples/Physics_applications/capacitive_discharge/PICMI_inputs_1d.py

```diff
@@ -208,11 +208,16 @@ def setup_run(self):
         self.sim.add_diagnostic(field_diag)
 
     def _get_rho_ions(self):
+        if self.rho_wrapper is None:
+            self.rho_wrapper = fields.RhoFPWrapper(0, False)
+
+        # clear rho_fp
+        self.rho_wrapper[Ellipsis] = 0.0
```

Comment:
Have not seen before that `Ellipsis` is an actual Python keyword for `...`. Wouldn't one just write
```suggestion
        self.rho_wrapper[...] = 0.0
```
for the Ellipsis, usually? :)

Reply:
Yes, I always use `[...]`. Less typing.

Reply:
Either one works (https://www.geeksforgeeks.org/what-is-three-dots-or-ellipsis-in-python3/).